experimental/devices/mfrc522/commands/registers.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package commands
+
+const (
+	Reserved00    = 0x00
```

Comment:
Do not export the reserved values? In fact I don't see the value in having these at all.